### ISSUE TITLE
Support for js source maps

### ISSUE THREAD

**Issue by polmuz** [django-compressor/django-compressor#438]
Source maps are a very nice tool for debugging something on compressed code using Chrome Dev Tools. There are lots of tools that generate them including Google closure that is already supported by django-compressor.

Here's some info: https://github.com/ryanseddon/source-map/wiki/Source-maps%3A-languages,-tools-and-other-info

**Comment by vladlep** [comment]
+1 I would like this feature also. It is very handy to be able to debug JS compressed files in production. Sentry errors do not make much sense if not.

**Comment by Woile** [comment]
+1 for the feature, absolutely required for Sentry errors :)

**Comment by wtayyeb** [comment]
+1 for js and css source maps

**Comment by ace-han** [comment]
I would like to see  js/ES6/coffee(even TypeScript) and css/sass/less source map
🍻

**Comment by travijuu** [comment]
Does Compress still not support source maps feature?

**Comment by diox** [comment]
No, it still does not support that feature. You're welcome to submit a PR implementing this.

**Comment by imposeren** [comment]
It's still possible to implement some "hackish" solution, for example:
https://bitbucket.org/imposeren/tabletop-planning/src/7d551806cfee2c5493b7273e40c5448aec480544/tabletop_tools/compressor_filters.py?at=master

Better solution is required to make PR...

**Comment by Andrioden** [comment]
For esbuild you can use inline sourcemap, and it works with django-compressor

``` js
await esbuild.build({
    // ...
    sourcemap: "inline",
})
```
